parsons/freshdesk/freshdesk.py

```diff
@@ -104,7 +104,7 @@ def get_tickets(self, ticket_type=None, requester_id=None, requester_email=None,
         return self.transform_table(tbl, expand_custom_fields)
 
     def get_contacts(self, email=None, mobile=None, phone=None, company_id=None,
-                     state=None, updated_since=None, expand_custom_fields=None):
+                     state=None, _updated_since=None, expand_custom_fields=None):
```

Comment:
I would say keep the Parsons parameter as `updated_since`. The underscore is a peculiarity of the FreshDesk API that we can hide from the Parsons user in our parameter naming.